src/TableColumn/Multiformat.php

```diff
@@ -0,0 +1,80 @@
+<?php
```

Comment:
Please add at least some type-hint comments for properties and methods :)

Reply:
adding..